packages/serverpod/lib/src/database/database.dart

```diff
@@ -192,6 +192,24 @@ class Database {
     );
   }
 
+  /// Executes a single SQL query. A [List] of rows represented of a [Map] with
+  /// the table name and value is another [Map] with the keys as column names and
+  /// the value as the contents of the column.
+  /// You are responsible to sanitize the query to avoid SQL injection.
+  Future<List<Map<String, Map<String, dynamic>>>> mappedResultsQueryDangerously(
```

Comment:
Suggestion: By using dangerously as a prefix it helps group the methods together and marks the most important aspect of them first.

```suggestion
  Future<List<Map<String, Map<String, dynamic>>>> dangerouslyQueryMappedResults(
```

Reply:
I would suggest reflecting this change for the other dangerously method. `queryDangerously`. -> `dangerouslyQuery`

Reply:
alright!